pkg/engine/virtualmachinescalesets.go

```diff
@@ -688,6 +688,7 @@ func CreateAgentVMSS(cs *api.ContainerService, profile *api.AgentPoolProfile) Vi
 		}
 
 		if cs.Properties.IsHostedMasterProfile() {
+			aksBillingExtension.Name = to.StringPtr(fmt.Sprintf("[concat(variables('%sVMNamePrefix'), '-AksLinuxBilling')]", profile.Name))
```

Comment:
is this going to affect how we track billing data in any way?

`Aks` should be `AKS`

What about Windows? This name contains "linux" and will be applied to all extensions, even Windows ones so it might be confusing.

Reply:
I see, before the extension name was `computeAksLinuxBilling`. This might break how we track these extensions if we query by name `computeAksLinuxBilling`.

Reply:
Actually before the extension name was set to <vmssname>-computeLinuxBilling, e.g. aks-nodepool1-40022152-vmss-computeAksLinuxBilling. Don't think this name that varies on different VMSS is used to track billing data.

Reply:
good point. @CecileRobertMichon we'd like to get this merged since this broke existing VMSS clusters. If billing tracking is broken, we take the risk.